client/src/app/components/Autocomplete.tsx

```diff
@@ -0,0 +1,284 @@
+import React, { useState, useRef } from "react";
+import {
+  Label,
+  LabelProps,
+  Flex,
+  FlexItem,
+  Menu,
+  MenuContent,
+  MenuItem,
+  MenuList,
+  Popper,
+  SearchInput,
+  Divider,
+} from "@patternfly/react-core";
+
+export interface IAutocompleteProps {
+  options?: string[];
+  placeholderText?: string;
+  searchString?: string;
+  searchInputAriaLabel?: string;
+  labelColor?: LabelProps["color"];
+  selections?: Set<string>;
+  menuHeader?: string;
+  noResultsMessage?: string;
+}
+
+export const Autocomplete: React.FC<IAutocompleteProps> = ({
+  // TODO: data just for testing purposes, should be removed
```

Comment:
Could you implement this in a field for testing purposes as a part of this PR?

Reply:
@ibolton336 - it's being used for tags field now